### PR TITLE
[chore](recycler) Change tablet_rowset_stale_sweep_time_sec=600 to reduce possibility failed to find rowset

### PR DESCRIPTION
**Pull request by gavinchou**
When there is a large query, the stale rowsets may be recycled before the query referencing them, which may cause -230, or import of MoW table failed to find rowset.

### CODE REVIEW

**Comment by gavinchou** [comment]
run buidlall

**Comment by gavinchou** [comment]
run buildall

**Comment by gavinchou** [comment]
run buildall

**Review by bobhan1 — APPROVED** [review]

**Review by liaoxin01 — APPROVED** [review]
LGTM

File: be/src/common/config.cpp [hunk 1]
```diff
@@ -296,7 +296,7 @@ DEFINE_mInt32(default_num_rows_per_column_file_block, "1024");
 // pending data policy
 DEFINE_mInt32(pending_data_expire_time_sec, "1800");
 // inc_rowset snapshot rs sweep time interval
-DEFINE_mInt32(tablet_rowset_stale_sweep_time_sec, "300");
+DEFINE_mInt32(tablet_rowset_stale_sweep_time_sec, "600");
```
Comment (gavinchou) [inline be/src/common/config.cpp:299]: LGTM

**Pull request merged**